### ISSUE TITLE
Add network --host to build script

### ISSUE THREAD

**Issue by mrakitin** [NSLS-II/lightsource2-recipes#632]
First confirm with group this is okay (in a pr to recipes)

**Comment by mrakitin** [comment]
➤ Julien Lhermitte commented:

This was an issue I had here when playing on thuban:
Testing on thuban - packaging and deployment ( https://3.basecamp.com/3733838/buckets/4534819/messages/1114837477 )

I noticed that there was some issue with the network bridge. When I removed it by adding the "--network host" flag, everything worked fine. If there are issues with docker not being able to see the outside world, then I suggest adding that flag here:

https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/nsls2-tag-build.sh#L36

I don't remember more than this and don't have time to try to recall so I will leave it at that. [@Maksim](https://3.basecamp-static.com/3733838/people/BAhpA50C6w==--a7b52001234d7899ebe013271da26cea6bdce557/avatar-20-x1?v=1) has taken charge of the build sytem so i will assign him and decide if this is needed
